templates/server/src/main/resources/archetype-resources/core/pom.xml

```diff
@@ -252,6 +252,17 @@
           </excludes>
         </configuration>
       </plugin>
+      <plugin>
+		<groupId>org.springframework.boot</groupId>
```

Comment:
Sorry for being picky, but can you format this and fix the indentation.
If you are using latest devonfw-ide with eclipse all should be preconfigured and you only need to format (`[shift][ctrl][F]`).